components/button/button.css

```diff
@@ -0,0 +1,67 @@
+:root {
+    --btn-color: #059669;
+    --btn-danger-color: red;
+    --btn-orange-color: orange;
+}
+
+button {
+    cursor: pointer;
+}
+
+.button {
```

Comment:
if you are using class names like btn-primary, its better to keep btn class instead of button to maintain consistency.